instrumentation-tools/nightly_tests_code_coverage.py

```diff
@@ -0,0 +1,94 @@
+#  Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+#  This product includes software developed at Datadog (https://www.datadoghq.com/).
+#  Copyright 2016-Present Datadog, Inc.
+
+import sys
+from argparse import ArgumentParser, Namespace
+from src.feature import Feature
+from src.api_surface import ApiSurface
+
+API_SURFACE_PATH = "dd-sdk-android/apiSurface"
+NIGHTLY_TESTS_DIRECTORY_PATH = "instrumented/nightly-tests/src/androidTest/kotlin"
+NIGHTLY_TESTS_PACKAGE = "com/datadog/android/nightly"
+IGNORED_GROUPS_REG_EX = "^(errorevent|viewevent|resourceevent|actionevent|" \
```

Comment:
The problem is that the prefix can be longer, it can start with the `LogEvent` but it can be `LogEvent#NetworkInfo#Carrier` depending on how many subclasses you have in there.

Reply:
I will add an explanation there